settings/EmailFiling.setting.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/**
+ * @file
+ * Email Filing Setting file.
+ */
+
+$settings = [
```

Comment:
Since we have chosen to go the route of adding the setting field with pre and post hooks, we don't need this file anymore and we can remove it.